src/main/scala/com/gu/configraun/models/Stage.scala

```diff
@@ -1,6 +1,12 @@
 package com.gu.configraun.models
 
-sealed trait Stage { def name: String }
+sealed trait Stage {
+  def name: String
+  def apply(value: String): Option[Stage] = {
+    Vector(DEV, CODE, PROD).find(_.toString == value)
```

Comment:
Why do you use a Vector here rather than a `Seq` or `Set` even given each value is unique?

Reply:
Because I nicked the code off StackOverflow of course.  Set would be conceptually better.  I'll change.